### ISSUE TITLE
Added new error code - build failed

### ISSUE THREAD

**Issue by per-samuelsson** [Starcounter/Starcounter.ErrorCodes#23]
https://teamcity.starcounter.org/viewLog.html?tab=buildLog&logTab=tree&filter=debug&expand=all&buildId=31262#_focus=248

What's the recommended practice when this happen? Just trigger a new build explicitly? That's what I'll try anyway. 😎

**Comment by per-samuelsson** [comment]
> Just trigger a new build explicitly? That's what I'll try anyway.

Didn't help, same result: https://teamcity.starcounter.org/viewLog.html?tab=buildLog&logTab=tree&filter=debug&expand=all&buildId=31263#_focus=75

**Comment by chrhol** [comment]
Never seen this before. It seems that some dll is locked. I will try it locally.

**Comment by chrhol** [comment]
And to answer the first question: adding a code should be enough. Builds will be triggered automatically and a new package pushed to myget.

**Comment by chrhol** [comment]
I get the same error building locally so something have changed. Investigating.

**Comment by per-samuelsson** [comment]
> I will try it locally.

I really like the current simplicity you've set up there: just push a new code and you'll get a new version of the package.

But maybe it would be good to somehow also document how to build and test a certain code locally. I tried finding a solution, but didn't. I guess you use VS code or smth? Can we just add a few lines on that? Like: 

> if you want to build, run `dotnet build` in this folder, and you should see `Error.cs` in this folder and the assembly in this folder yada yada

**Comment by chrhol** [comment]
There is a new version of errorcodes on myget now.

It looks like each target framework is built in parallel (which I guess it didn't do earlier since the builds succeeded), and the generator project is called for each which make some file being locked. I did a quick fix and limited the number of allowed cores to 1 to get the build to succeed. 

It's not a big prio, but we probably want to redo how/when the generator is triggered to allow building in parallel, especially since you need to limit the number of cores when building locally to. I will add a new issue for that.

> But maybe it would be good to somehow also document how to build and test a certain code locally.

Alright, I will clarify the readme with some steps on what actually happens when you build, and what you can do if you want to test locally.

**Comment by per-samuelsson** [comment]
> There is a new version of errorcodes on myget now.

Thanks @chrhol. Considering this done.
